src/main/java/uk/org/tombolo/importer/ons/ONSClaimantsImporter.java

```diff
@@ -64,6 +64,11 @@ public DatasourceSpec getDatasourceSpec(String datasourceIdString) throws Except
         return DatasourceId.valueOf(datasourceIdString).datasourceSpec;
     }
 
+    @Override
+    protected List<String> getOaDatasourceIds() {
+        return Collections.singletonList(OaImporter.OaType.lsoa.datasourceSpec.getId());
+    }
```

Comment:
wont this restrict user to only work with lsoa boundaries?

Reply:
It would be good, if we could have a recipe that tests this.

Reply:
Please see the general comment.